apps/api-catalogue/components/HomeLayout/HomeLayout.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactNode } from 'react'
+import { useWindowSize, useIsomorphicLayoutEffect } from 'react-use'
+import { Box, ContentBlock, Columns, Column } from '@island.is/island-ui/core'
+import { theme } from '@island.is/island-ui/theme'
+
+interface PropTypes {
+  left: ReactNode
+  right?: ReactNode
+}
+
+function HomeLayout({ left, right }: PropTypes) {
+  const { width } = useWindowSize()
+  const [isMobile, setIsMobile] = React.useState(false)
+
+  useIsomorphicLayoutEffect(() => {
+    //if (width < 771) {
+    if (width < theme.breakpoints.md) {
+      return setIsMobile(true)
+    }
+    setIsMobile(false)
+  }, [width])
+
+  return (
+    <Box paddingX="gutter">
+      <ContentBlock>
+        <Columns align="right" space="gutter">
+          <Column width={isMobile ? '11/12' : '6/12'}>{left}</Column>
```

Comment:
Good point. The `width` prop should definitely be responsive.

Reply:
I will add that functionality.

Reply:
I see now that you are using `Columns` for the actual page layout. Please check out `Grid` component for that. We will consider renaming `Columns` and `Column` to prevent this misunderstanding.